kratos/modeler/voxel_mesh_generator_modeler.cpp

```diff
@@ -0,0 +1,332 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/mesh.h"
+#include "includes/element.h"
+#include "includes/model_part.h"
+#include "geometries/bounding_box.h"
+#include "geometries/point.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/timer.h"
+#include "spatial_containers/geometrical_objects_bins.h"
+#include "utilities/model_part_utils.h"
+
+#include "voxel_mesh_generator_modeler.h"
+#include "coloring/voxel_mesher_coloring_factory.h"
+#include "key_plane_generation/key_plane_generation_factory.h"
+#include "entity_generation/voxel_mesher_entity_generation_factory.h"
+#include "operation/voxel_mesher_operation_factory.h"
+
+namespace Kratos
+{
+using SizeType = std::size_t;
```

Comment:
std::size_t is being used directly in the file in most of the cases. Unless the implementation for some code is not dependent on the size I would just remove this.